test/e2e/Services/DockerService.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { DockerService } from '../../../src/main/Services/DockerService';
+import { DockerRunParams } from '../../../src/main/Utils/Docker/DockerRunParams';
+import { runCmd } from '../../../src/main/Utils/cmd-utils';
+import { DockerCommand } from '../../../src/main/Utils/Docker/DockerCommand';
+
+describe('Docker service e2e tests', () => {
+    let dockerService: DockerService;
+    const dockerImageParams: DockerRunParams = {
+        image: 'alpine',
+        name: 'test-image',
+        detached: true,
+        cmd: 'tail -f /dev/null'
+    };
+
+    beforeEach(async () => {
+        dockerService = new DockerService();
+        await dockerService.start();
```

Comment:
we agreed not to make docker e2e test run if there is no docker installed

Reply:
@mpetrunic, for now, I will solve this with if statement in these few tests, but I will investigate some wrapper to get test_if functionality with the async condition.

Reply:
You should be able to set skip in before clause

Reply:
but yeah whatever